### PR TITLE
Fixed missing separation lines

### PR DESCRIPTION
**Pull request by ghost**
I managed to get it working for gtk-2.0 and gtk-3.20 if someone can do the others I'd be happy.

### CODE REVIEW

File: common/gtk-3.0/3.20/sass/_common.scss [hunk 1]
```diff
@@ -1601,6 +1601,7 @@ menubar,
 menu,
 .menu {
   $_menu_bg: if($variant=='light', $base_color, $bg_color);
+  $_sp_menu_bg: #dfdfdf;
```
Comment (NicoHood) [inline common/gtk-3.0/3.20/sass/_common.scss]: This should get a color from the predefined colors in colors.scss and not a fixed value like this. It also have to work with the dark theme.

File: common/gtk-3.0/3.20/sass/_common.scss [hunk 2]
```diff
@@ -1616,18 +1617,18 @@ menu,
...
 
   // Firefox workaround
   .separator:not(label),
-  .csd & .separator:not(label) { color: $_menu_bg; }
+  .csd & .separator:not(label) { color: $_sp_menu_bg; }
   // Firefox workaround end
 
   menuitem {
-    min-height: 16px;
+    min-height: 13px;
```
Comment (NicoHood) [inline common/gtk-3.0/3.20/sass/_common.scss]: The color change above makes sense to me, but could you explain in short why to also change the height and padding?

File: common/gtk-2.0/main.rc [hunk 3]
```diff
@@ -31,7 +31,7 @@ style "default" {
   GtkRange::stepper-size = 0
 
   GtkScrollbar::activate-slider = 1
-  GtkScrollbar::has-backward-stepper = 0  
+  GtkScrollbar::has-backward-stepper = 0
```
Comment (NicoHood) [inline common/gtk-2.0/main.rc]: Next time it would be better to split whitespace fixes in a single comment. I usually safe the file (my editor autoremoves them), commit the changes and then do my fixes. Its okay for this minor change, but for larger PRs its harder to review. You possibly already knew that, I am lazy too sometimes :D

File: common/gtk-2.0/main.rc [hunk 4]
```diff
@@ -1949,13 +1949,11 @@ style "progressbar" {
...
 style "separator_menu_item" {
-
   engine "pixmap" {
-
     image {
       function = BOX
-      file = "assets/null.png"
-      border = { 0, 0, 1, 0 }
+      file = "assets/line-h.png"
+      border = { 20, 0, 3, 0 }
```
Comment (NicoHood) [inline common/gtk-2.0/main.rc]: Just to understand your changes:
You made the actual space between two sections smaller, but added a horizontal line for it instead, right?

Could you please explain the values 20,0,3,0 in short? (I am also learning gtk, would be nice if you can tell me why you've chosen those).

Now I am wondering if horst did the extra space (instead of lines) by intention. However from your screenshot it looks okay for the light theme, but we need to test it for the dark theme as well. Can you please test this and also add a screenshot for darker and dark variant?

Comment (iJackTheRipper) [reply common/gtk-2.0/main.rc]: I have no clue why 20,0,3,0, but it just works

File: common/gtk-3.0/3.20/sass/_common.scss (hunk 1 above)
Comment (iJackTheRipper) [reply common/gtk-3.0/3.20/sass/_common.scss]: Yes I know it needs more work, but I think you guys will improve it.

File: common/gtk-3.0/3.20/sass/_common.scss (hunk 2 above)
Comment (iJackTheRipper) [reply common/gtk-3.0/3.20/sass/_common.scss]: Because it wasn't in the middle it has more padding on top for some reason I don't get it.

**Comment by iJackTheRipper** [comment]
It just needs a lot of work and to generate the CSS from SASS takes so long

**Comment by NicoHood** [comment]
@fossfreedom If you can help with your knowledge, let us know. I will try to further test this the next weeks. @iJackTheRipper Thanks for this first fix!

**Comment by iJackTheRipper** [comment]
Just a sec

File: common/gtk-3.0/3.20/sass/_colors.scss [hunk 5]
```diff
@@ -6,11 +6,18 @@
 }
 
 $base_color: if($variant =='light', #ffffff, #404552);
+
+$text_color: if($variant == 'light', #5c616c, #D3DAE3);
+$bg_color: if($variant =='light', #F5F6F7, #383C4A);
+$fg_color: if($variant =='light', #5c616c, #D3DAE3);
+$sp_menu: if($variant =='light', #dfdfdf, #2b2e39);
+
```
Comment (jnsh) [inline common/gtk-3.0/3.20/sass/_colors.scss]: Why are `$text_color` `$bg_color` and `$fg_color` declared here, while they are reset with original values right below?
If there's some purpose for this, why did you change the original color values for the light variant?

Also, I think lines `9`, `14` and `20` are unnecessary whitespace.
